canonical/charmhub.io
# fix: interface pages after repo modification

## Done
- Refactor of the way we fetch a single interface - focusing on a single item based on the directory structure and embedded metadata.

## How to QA
Visit any charm and click through interfaces. 
- demo/interfaces/nonexistent_interface should 404
- demo/interfaces/prometheus_scrape should show "featured" charms
- demo/interfaces/auth_proxy should redirect to /interfaces/auth_proxy/draft
- demo/interfaces/tls-certificates should show a list of charms, but no documentation
- demo/interfaces/tls_certificates should show documentation, but no charms

## Issue / Card
Fixes https://warthogs.atlassian.net/browse/WD-10915
